pkg/application/vault/README.md

```diff
@@ -0,0 +1,108 @@
+# `Hashicorp Vault` - Swiss Army Knife of key-value Stores
+
+## Documentation
+[Kubernetes Deployment Guide](https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-raft-deployment-guide)
+[EKS-Specific Guide](https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-amazon-eks)
+
+## Install with EKSDemo
+`eksdemo install vault`
+
+## Installation Arguments
+- `enable-tls` -> Enable TLS for end-to-end encrypted transport
+  - Diabled by default
```

Comment:
don't forget to remove this from the documentation as you pulled it out of the PR